pygmt/tests/test_basemap.py

```diff
@@ -54,15 +56,17 @@ def test_basemap_power_axis():
     return fig
 
 
-@pytest.mark.xfail(
-    reason="Baseline image not updated to use earth relief grid in GMT 6.1.0",
-)
-@pytest.mark.mpl_image_compare
+@check_figures_equal()
 def test_basemap_polar():
     "Create a polar basemap plot"
-    fig = Figure()
-    fig.basemap(R="0/360/0/1000", J="P6i", B="afg")
-    return fig
+    fig_test = Figure()
+    fig_test.basemap(R="0/360/0/1000", J="P6i", B="afg")
+
+    fig_ref = Figure()
+    with Session() as lib:
+        lib.call_module("basemap", "-R0/360/0/1000 -JP6i -Bafg")
```

Comment:
Just as a throwaway idea, what about testing that the long aliases and list-to-str conversion works? I.e. have `fig_test` use long aliases like `region`, and have `fig_ref` use the short alias like `R`.

```suggestion
    fig_test = Figure()
    fig_test.basemap(region=[0, 360, 0, 1000], projection="P6i", frame="afg")

    fig_ref = Figure()
    fig_ref.basemap(R="0/360/0/1000", J="P6i", B="afg")
```

Reply:
Sounds a clever idea. I believe we are confident that arguments like `R="0/360/0/1000", J="P6i", B="afg"` will be converted to `"-R0/360/0/1000 -JP6i -Bafg"`, thus, `fig_ref.basemap(R="0/360/0/1000", J="P6i", B="afg")` is the same as `lib.call_module("basemap", "-R0/360/0/1000 -JP6i -Bafg")`.

Reply:
Cool, if you want to update this PR to fix the failing 'polar', 'logo' (as mentioned at https://github.com/GenericMappingTools/pygmt/issues/451#issuecomment-708792970) tests, and any other ones that pop up (I saw a few more upstream GMT changes already...), I'll review it later.

Reply:
OK. I'll work on this PR.